prisma/schema.prisma

```diff
@@ -127,3 +130,87 @@ model Stage {
   name        String @unique
   areas       Area[]
 }
+
+// Added tables for Assessment
+
+model Project {
+  id            Int         @id @default(autoincrement())
+  name          String
+  description   String
+  archived      Boolean
+  participants  User[]      @relation("UserProjects")
+  created_at    DateTime    @default(now())
+  assessments   Assessment[]
+}
+
+model Assessment {
+  userId                    String
+  projectId                 Int
+  user                      User       @relation(fields: [userId], references: [id])
+  project                   Project    @relation(fields: [projectId], references: [id])
+  areasMaturityScore        Float
+  artefactCompletenessScore Float
+  areaCompletenessScore     Float
+  answersArea               AnswerAreaQuestion[]
+  answersArtefact           AnswerArtefact[]
+  stagesScores              StageScore[]
+  areasScores               AreaScore[]
+
+  @@id([userId, projectId])
+}
+
+model StageScore {
+  id                            Int       @id @default(autoincrement())
+  stageId                       Int
+  assessmentId                  Int
+  artefactsCompletenessScore    Float
+  areaMaturityScore             Float
+  areaCompletenessScore         Float
+  targetAreaMaturityScore       Float
+  stage                         Stage     @relation(fields: [stageId], references: [id])
+  assessment                    Assessment @relation(fields: [assessmentId], references: [userId, projectId])
+}
+
+model Question {
+  id                  Int             @id @default(autoincrement())
+  body                String
+  areas               Area[]
+  answerAreaQuestions AnswerAreaQuestion[]
+}
+
+model AnswerAreaQuestion {
+  id            Int         @id @default(autoincrement())
+  questionId    Int
+  assessmentId  Int
+  assessedScore Int
+  targetScore   Int
+  answered      Boolean
+  comment       String
+  created_at    DateTime    @default(now())
```

Comment:
Why do we need a time stamp? In this case, would an additional time stamp `last_modified` make sense?